thirdeye/thirdeye-frontend/app/pods/home/route.js

```diff
@@ -1,4 +1,77 @@
 import Route from '@ember/routing/route';
+import applicationAnomalies from 'thirdeye-frontend/mocks/applicationAnomalies';
+import columns from 'thirdeye-frontend/shared/anomaliesTableColumns';
 
 export default Route.extend({
+
+  /**
+   * Returns a two-dimensional array, which maps anomalies by metric and functionName (aka alert)
+   * @example
+   * [
+   *  "Metric 1": [
+   *    "Alert 1": [ {anomalyObject}, {anomalyObject} ],
+   *    "Alert 11": [ {anomalyObject}, {anomalyObject} ]
+   *  ],
+   *  "Metric 2": [
+   *    "Alert 21": [ {anomalyObject}, {anomalyObject} ],
+   *    "Alert 22": [ {anomalyObject}, {anomalyObject} ]
+   *   ]
+   * ]
+   */
+  model() {
+    let anomalyMapping = [];
+
+    applicationAnomalies.forEach(anomaly => {
+      const { metric, functionName, current, baseline } = anomaly;
+      const value = {
+        current,
+        baseline
+      };
+
+      if (!anomalyMapping[metric]) {
+        anomalyMapping[metric] = [];
```

Comment:
I'm confused. metric and Function names seem to be strings, and anomalyMapping is an array, but you're using them to create new arrays as properties on your mapping Array? And then functionName is an array on the array you created on the array?

Reply:
+1 I think this should be {} instead

Reply:
Gotcha. Thanks for the comments! Will change this to a `{}` instead.